rotkehlchen/api/rest.py

```diff
@@ -4495,3 +4496,29 @@ def get_config_arguments(self) -> Response:
             },
         }
         return api_response(_wrap_in_ok_result(config), status_code=HTTPStatus.OK)
+
+    def get_user_notes(self) -> Response:
+        user_notes = self.rotkehlchen.data.db.get_all_user_notes()
+        result_dict = _wrap_in_ok_result([entry.serialize() for entry in user_notes])
+        return api_response(result_dict, status_code=HTTPStatus.OK)
+
+    def add_user_note(self, title: str, content: str, location: str) -> Response:
```

Comment:
The reason I didn't pass `UserNote` here was because I'd have to make two fields optional i.e `identifier` and `creation_timestamp`. During creation, these fields are unknown.

If you have a better approach as to how to make it better, lemme know

Reply:
it's fine, keep it as is